layouts/partials/docs-nav.html

```diff
@@ -36,6 +36,61 @@
   {{ .Scratch.Set "path" $path}}
 {{ end }}
 
+{{ define "navChildList" -}}
+{{ $item := .item -}}
+{{ $currentPageRelRef := .currentPageRelRef -}}
+{{ $hidden := not ( hasPrefix $currentPageRelRef $item.relref )}}
+<ul class="child-list{{ if $hidden }} hidden{{ end }}">
+  {{ range $item.children }}
+  {{ $isActive := hasPrefix $currentPageRelRef .relref }}
+  {{- if .divider }}
+  <li class="my-2 bg-redis-pencil-200"><strong><span class="pr-5"></span><a href='{{ page.RelRef (dict "path" .relref) }}'> {{ .linkTitle }}</a></strong>
+  {{- else if .children }}
+  <li class="my-2">
+    <a href='{{ page.RelRef (dict "path" .relref) }}' class="pl-6 flex{{ if $isActive }} active text-redis-ink-900 {{ else }} text-redis-pen-700 hover:text-redis-pen-600 {{ end }}" style="pointer-events: all;"><span class="pr-5"></span>{{ .linkTitle }}<span class="pr-2"></span>
+      <span class="arrowBox">
+        <svg style="display: inline;" width="18" height="18" version="1.1" xmlns="http://www.w3.org/2000/svg">
+          <polyline points="3 8 15 8 9 3" fill="black" />
+          <polyline points="3 10 9 15 15 10" fill="black" />
+        </svg>
+      </span>
+    </a>
+  {{- template "navChildList" ( dict "item" . "currentPageRelRef" $currentPageRelRef )}}
+  {{- else }}
+  <li class="my-2"><a href='{{ page.RelRef (dict "path" .relref) }}' class="pl-6 flex{{ if $isActive }} active text-redis-ink-900 {{ else }} text-redis-pen-700 hover:text-redis-pen-600 {{ end }}" style="pointer-events: all;"><span class="pr-5"></span>{{ .linkTitle }}</a>
+  {{- end -}}
+  </li>
+  {{ end }}  
+</ul>
+{{- end }}
+
+{{ define "navList" -}}
+{{ $root := .root -}}
+{{ $currentPage := $.currentPage }}
+<ul class="md:block overflow-y-auto grow pr-0  border-opacity-50 border-redis-ink-900">
+  {{ range $root.children -}}
+  {{ $isActive := hasPrefix $currentPage.RelPermalink .relref }}
+  {{- if .divider }}
+  <li class="my-2 bg-redis-pencil-500 text-white text-center"><span class="pr-5"></span><strong><a href='{{ page.RelRef (dict "path" .relref) }}'>{{ .linkTitle }}</a></strong>
```

Comment:
```suggestion
  <li class="my-2 bg-redis-pencil-500 text-white text-center rounded-r-md"><span class="pr-5"></span><strong><a href='{{ page.RelRef (dict "path" .relref) }}'>{{ .linkTitle }}</a></strong>
```

Reply:
@cmilesb I like that - it seems to emphasise the meaning of the divider. I've tried adding rounded corners at both ends. What do you think?

Reply:
@andy-stark-redis I personally prefer it with just the right corners rounded. With both sets of corners rounded it looks weird since the left side hits right against the menu's border.